### DISCUSSION TITLE
86BOX display remains blank but audio works.

### DISCUSSION THREAD

**Discussion by KoghaandSooga** [86Box/86Box#3667] [Q&A] [answered]
Everytime I install an OS on 86box and turn it off it will never show up again and remain a blank display meaning you can only use it one time and can never do so again.  I'm using the latest build b4755.   I'm not a programmer so it's way out of my expertise but I'm generally good with setting up various emulators and I've not had this trouble on PCEM V17 and this is the only place to post as I can't handle chat rooms.

**Comment by TC1995** [comment]
Latest is not 4755.

**Comment by jriwanek** [comment, marked as answer]
Relatedly, we have a template for a reason, and you should fill it out, not remove it. Closing as you didn't.
(Also, it should have been an issue, not a discussion)
